bin/steps/pip-install

```diff
@@ -38,10 +38,10 @@ if [ ! "$SKIP_PIP_INSTALL" ]; then
         mcount "failure.none-version"
     fi
 
-    if grep -q 'django==1.*' requirements.txt; then
+    if grep -qi '^django==1.*' requirements.txt; then
```

Comment:
I believe this is the "ignore case" flag, right? Is there a reason this needs to be added?

Reply:
From talking with @CaseyFaist, here's the [PEP](https://www.python.org/dev/peps/pep-0426/#name).